### ISSUE TITLE
The result about Top-N precision

### ISSUE THREAD

**Issue by YangLi1221** [thunlp/HNRE#4]
I run your pretrained model to get the result of pone, ptwo and pall. But all of them had a great gap with the results given in your paper. 
How to solve this problem?

**Comment by lulu51230** [comment]
I also encountered the same situation as you. Can you solve it?

**Comment by lulu51230** [comment]
I also encountered the same situation as you. Can you solve it?

**Comment by YangLi1221** [comment]
no

**Comment by furkanozbay** [comment]
Same problem here

**Comment by lulu51230** [comment]
hi,@furkanozbay,please,how to solve it?what the mean about same problem here?

**Comment by furkanozbay** [comment]
hi @lulu51230 I have the same problem and I don't know how to solve it. Both pretrained and my trained models' scores are lower than expected scores. Maybe epoch count can improve the performance

**Comment by Perfec-Yu** [comment]
Hi @furkanozba. Do you have problems with PCNN model? Indeed it is a little bit difficult to train that model and with my code here there could be a very small gap with the reported PCNN model in MAP. The known issue about p@N here is that I found the PCNN PALL P@100 seems to be somewhat lower. But all the other figures should be fine. Can you give more details about your experiments ?

**Comment by furkanozbay** [comment]
Hi @Perfec-Yu 
When I test pre-trained pcnn model which is stored in outputs folder with pall method, the output is 0.82,  0.73,  0.68 ( expected results are "0.88, 0.79,  0.77")
and for cnn model pall method, the output is;   0.8,   0.70,   0.68  ( expected results are "0.88 0.79 0.75")

Also if train the cnn model; I can find top100 results at most 0.79

**Comment by THUCSTHanxu13** [comment]
Our released toolkit was reconstructed based on the original code. I have checked the code and found that the initialization code may lead to mistakenly dropping some data. We have fixed this problem, and the released checkpoints can achieve comparable results with our reported ones (As the released checkpoints were also retrained on this toolkit, the random operations in training and test make it difficult to achieve identical results).
